### ISSUE TITLE
训练检测模型后，eval.py输出的指标都是0，infer_det.py测试结果框出的也是全图，该怎么办

### ISSUE THREAD

**Issue by s957995299** [PaddlePaddle/PaddleOCR#4790]
检测模型的训练命令：python tools/train.py -c configs/det/det_mv3_db.yml -o Global.pretrained_model=./pretrain_models/MobileNetV3_large_x0_5_pretrained
检测模型的训练结果：
![image](https://user-images.githubusercontent.com/83860013/143808664-9d8433b4-638b-47c9-bf17-91929d74e8e4.png)
评估命令：python tools/eval.py -c configs/det/det_mv3_db.yml  -o Global.checkpoints="./output/db_mv3/best_accuracy" PostProcess.box_thresh=0.6 PostProcess.unclip_ratio=1.5
评估结果：
![image](https://user-images.githubusercontent.com/83860013/143808781-79dd3575-2bbb-4b4f-b218-aca067d9b49b.png)
测试命令：python tools/infer_det.py -c configs/det/det_mv3_db.yml -o Global.infer_img="./doc/imgs_en/img_10.jpg" Global.pretrained_model="./output/db_mv3/best_accuracy"
测试结果：
![image](https://user-images.githubusercontent.com/83860013/143808957-81f8eea3-49db-423c-a99c-511999f81721.png)
![image](https://user-images.githubusercontent.com/83860013/143809019-9548f515-55ea-49e7-a34c-4327d664d967.png)

**Comment by WenmuZhou** [comment]
这个是你的模型没收敛哈，建议检查下数据

**Comment by s957995299** [comment]
> 这个是你的模型没收敛哈，建议检查下数据

我用官方给的icdar2015数据集训练的，您看这有什么问题么
![image](https://user-images.githubusercontent.com/83860013/144005089-4c2fa257-bc8a-4825-9747-4126465c7ea1.png)
配置文件如下
Global:
  use_gpu: true
  epoch_num: 1200
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/db_mv3/
  save_epoch_step: 1200
  # evaluation is run every 2000 iterations after the 0th iteration
  eval_batch_step: [0, 2000]
  cal_metric_during_train: False
  pretrained_model: ./pretrain_models/MobileNetV3_large_x0_5_pretrained
  checkpoints:
  save_inference_dir:
  use_visualdl: False
  infer_img: doc/imgs_en/img_10.jpg
  save_res_path: ./output/det_db/predicts_db.txt

Architecture:
  model_type: det
  algorithm: DB
  Transform:
  Backbone:
    name: MobileNetV3
    scale: 0.5
    model_name: large
  Neck:
    name: DBFPN
    out_channels: 256
  Head:
    name: DBHead
    k: 50

Loss:
  name: DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 3

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    learning_rate: 0.001
  regularizer:
    name: 'L2'
    factor: 0

PostProcess:
  name: DBPostProcess
  thresh: 0.3
  box_thresh: 0.6
  max_candidates: 1000
  unclip_ratio: 1.5

Metric:
  name: DetMetric
  main_indicator: hmean

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/icdar2015/text_localization/
    label_file_list:
      - ./train_data/icdar2015/text_localization/train_icdar2015_label.txt
    ratio_list: [1.0]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - IaaAugment:
          augmenter_args:
            - { 'type': Fliplr, 'args': { 'p': 0.5 } }
            - { 'type': Affine, 'args': { 'rotate': [-10, 10] } }
            - { 'type': Resize, 'args': { 'size': [0.5, 3] } }
      - EastRandomCropData:
          size: [640, 640]
          max_tries: 50
          keep_ratio: true
      - MakeBorderMap:
          shrink_ratio: 0.4
          thresh_min: 0.3
          thresh_max: 0.7
      - MakeShrinkMap:
          shrink_ratio: 0.4
          min_text_size: 8
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'threshold_map', 'threshold_mask', 'shrink_map', 'shrink_mask'] # the order of the dataloader list
  loader:
    shuffle: True
    drop_last: False
    batch_size_per_card: 8
    num_workers: 4
    use_shared_memory: False

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/icdar2015/text_localization/
    label_file_list:
      - ./train_data/icdar2015/text_localization/test_icdar2015_label.txt
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - DetResizeForTest:
          image_shape: [736, 1280]
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'shape', 'polys', 'ignore_tags']
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 1 # must be 1
    num_workers: 2
    use_shared_memory: False

**Comment by WenmuZhou** [comment]
发一下完整的训练log看看

**Comment by s957995299** [comment]
> 发一下完整的训练log看看

[train.log](https://github.com/PaddlePaddle/PaddleOCR/files/7633361/train.log)
麻烦了
应该是从[2021/11/26 13:22:06]开始的部分

**Comment by WenmuZhou** [comment]
从日志看，你的loss一直没变，数据你改过吗

**Comment by s957995299** [comment]
> 

没有 用的就是官网下载的icdar2015的数据集，1000张训练图片，500张测试图片

**Comment by WenmuZhou** [comment]
然后就改了一下配置里的路径就开始训练了？

**Comment by s957995299** [comment]
> 然后就改了一下配置里的路径就开始训练了？

 确实，我大前天又重新下了一遍数据集，然后重新训练了一边，前天得到的结果还是一样，真是颓了
![image](https://user-images.githubusercontent.com/83860013/144735434-1f278743-443d-49e6-a9c3-d4fedf642dbf.png)

**Comment by zhangyingying520** [comment]
> > 然后就改了一下配置里的路径就开始训练了？
> 
> 确实，我大前天又重新下了一遍数据集，然后重新训练了一边，前天得到的结果还是一样，真是颓了 ![image](https://user-images.githubusercontent.com/83860013/144735434-1f278743-443d-49e6-a9c3-d4fedf642dbf.png)

![image](https://user-images.githubusercontent.com/80818728/145324372-80d873c0-c7a3-49d7-b2e1-0b186348852b.png)
你这图片路径不读，都不存在

**Comment by zhangyingying520** [comment]
> > 然后就改了一下配置里的路径就开始训练了？
> 
> 确实，我大前天又重新下了一遍数据集，然后重新训练了一边，前天得到的结果还是一样，真是颓了 ![image](https://user-images.githubusercontent.com/83860013/144735434-1f278743-443d-49e6-a9c3-d4fedf642dbf.png)

好好看看数据集图片和标签的路径

**Comment by s957995299** [comment]
> 好好看看数据集图片和标签的路径

不是 之前有说过  是从[2021/11/26 13:22:06]开始的部分.....emmmm

**Comment by s957995299** [comment]
@WenmuZhou  麻烦管理删一下这个问题吧 没有探究的意义了 原因过于傻 这个问题留着会误导别人的

**Comment by Li-Asia-Edward** [comment]
eval model::  75%|█████████████████████████████████████████████████████▎                 | 3/4 [00:01<00:00,  1.53it/s]
[2023/06/21 09:53:52] ppocr INFO: metric eval ***************
[2023/06/21 09:53:52] ppocr INFO: precision:0
[2023/06/21 09:53:52] ppocr INFO: recall:0.0
[2023/06/21 09:53:52] ppocr INFO: hmean:0
[2023/06/21 09:53:52] ppocr INFO: fps:1.5771145207727977
这个是什么原因造成的啊，能请教一下吗？
